bucket/gobrew.json

```
@@ -0,0 +1,60 @@
+{
+    "version": "v1.9.7",
+    "description": "Go version manager, written in Go. Super simple tool to install and manage Go versions. Install go without root. Gobrew doesn't require shell rehash. ",
+    "homepage": "https://github.com/kevincobain2000/gobrew",
+    "license": "MIT",
+    "notes": [
+        "gobrew uses symlinks to switch between go versions.",
+        "To be able to create symlinks without elevated privileges, Developer Mode needs to be enabled.",
+        "You can do this either in the settings app, or by setting the HKLM\\Software\\Policies\\Microsoft\\Windows\\Appx\\AllowDevelopmentWithoutDevLicense RKey accordingly."
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/kevincobain2000/gobrew/releases/download/v1.9.7/gobrew-windows-amd64.exe",
+            "hash": "bb097b4cf10308ec89f41f5cb602d2a8acd349da8e08e0522d5b807c7ad9fa1a"
+        }
+    },
+    "installer": {
+        "script": [
+            "$envgopath = \"$env:USERPROFILE\\go\"",
+            "if ($env:GOPATH) { $envgopath = $env:GOPATH }",
+            "info \"Adding '$envgopath\\bin' to PATH...\"",
+            "add_first_in_path \"$envgopath\\bin\" $global"
+        ]
+    },
+    "env_add_path": [
+        ".gobrew\\current\\bin",
+        ".gobrew\\bin"
+    ],
+    "env_set": {
+        "GOBREW_ROOT": "$dir",
+        "GOROOT": "$dir\\.gobrew\\current\\go"
+    },
+    "bin": [
+        [
+            "gobrew-windows-amd64.exe",
+            "gobrew"
+        ]
+    ],
+    "persist": [
+        ".gobrew"
+    ],
+    "uninstaller": {
+        "script": [
+            "$envgopath = \"$env:USERPROFILE\\go\"",
+            "if ($env:GOPATH) { $envgopath = $env:GOPATH }",
+            "info \"Removing '$envgopath\\bin' from PATH...\"",
+            "remove_from_path \"$envgopath\\bin\" $global"
+        ]
+    },
+    "checkver": {
+        "github": "https://github.com/kevincobain2000/gobrew"
+    },
```

```suggestion
    "checkver": "github",
```

More laconic form since github link already used as `homepage` ([as per](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#special-cases)).